cassandra/encoder.py

```diff
@@ -217,3 +219,6 @@ def cql_encode_ipaddress(self, val):
         is suitable for ``inet`` type columns.
         """
         return "'%s'" % val.compressed
+
+    def cql_encode_decimal(self, val):
+        return self.cql_encode_float(float(val))
```

Comment:
Fixing a bug discovered during testing.  When using the drivers support for positional parameters in the execute() statement we weren't correctly expanding decimal types.